test/system/announcements_test.rb

```diff
@@ -125,7 +125,7 @@ class AnnouncementsTest < ApplicationSystemTestCase
 
     assert has_no_button? '公開'
     click_button '作成'
-    assert_text 'お知らせを作成しました'
+    assert page.has_css?('p.flash__message', text: 'お知らせを作成しました')
```

Comment:
```suggestion
    assert has_css?('p.flash__message', text: 'お知らせを作成しました')
```

同じくこちらもpageいらないかもです。他の箇所もチェックしてみてください〜

Reply:
@komagata 
すみません、確認漏れてました。
再度チェックし不要なpage削除しました。